index.html

```diff
@@ -0,0 +1,153 @@
+            <ul id="nav-items">
+              <li>
+                <a href="about.html" class="nav-links about-link">About</a>
+              </li>
+              <li>
+                <a href="#program" class="nav-links program-link" >Program</a>
+              </li>
+
+              <li>
+                <a href="#join" class="nav-links join-link">Join</a>
+              </li>
+
+              <li>
+                <a href="#sponsor" class="nav-links sponsor-link">Sponsor</a>
+              </li>
+
+              <li>
+                <a href="#news" class="nav-links news-link">News</a>
+              </li>  
+              
+              <li>
+                  <a href="#" class="nav-links active">AiT Summit</a>
+              </li>
+            </ul>
+                        
+              
+          </div>
+
+        </nav>  
+  </header>
+
+    <section class="landing-page">
+  <div class="landing-text">
+  <h3>"Hello! Sharing world"</h3>
+  <h1 class="title">TECH IN AFRICA GLOBAL SUMMIT 2022</h1>
+
+  <p>An insightful delebration on another bright spot in the history of the continent where there  is the continued improvement of  lives through tech, holding this April in Abuja Nigeria.</p>
+
+  <h4 class="date">2022.04.28(THU) ~ 29(FRI)</h4><br>
+  <h5 class="venue">@ Internation conference center, Abuja, Nigeria</h5>
+    </div>
+
+
+    <div class="main-program">
+
+        <h2>Main program</h2>
+        
+        <div class="main-wrap">
+        <div class="lecture item"><img src="images/lecture.png" alt="">Lecture <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fugiat, magnam.</p></div>
+        <div class="exibition item"><img src="images/exibition.jpg" alt="">Exibition<p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fugiat, magnam.</p></div>
+        <div class="forum item"><img src="images/forum.png" alt="">Forum<p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fugiat, magnam.</p></div>
+        <div class="workshop item"><img src="images/workshop.png" alt="">TiA workshop<p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fugiat, magnam.</p></div>
+        <div class="network item"><img src="images/ignite.png" alt="">TiA Ignite<p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Fugiat, magnam.</p></div>
+        </div>
+
+      <a href="#" class="join"> Join AiT Global summit 2022 </a>
+
+      <a href="#" class="program">See the Whole program</a>
+    </div>
+
+ 
+    </section>
+
+    <section class="speakers-list">
+       <h2>Featured Speakers</h2>
+    </section>
+
+     <section class="partners">
+        <h2>Partners</h2>
+
+        <hr />
+
+        <div class="partner1">
+          <a href="#"
+            ><img src="images/microverse.png" alt="Partner1" />Microverse</a
+          >
+
+          <a href="#"> <img src="images/google.jpg" alt="Partner1" />Google</a>
+
+          <a href="#">
+            <img src="images/mozilla.jpg" alt="Partner1" />mozilla</a
```

Comment:
- [ ] At the moment, the partners' section in the mobile view is not responsive and does not match the original design. Kindly make sure you stick to the design guideline as much as possible

### Your Design
![image](https://user-images.githubusercontent.com/43995830/163596936-a91d2cda-7271-4f1a-a0bb-5aece739b11e.png)


### Original Design
![image](https://user-images.githubusercontent.com/43995830/163596980-25308fab-3c28-4297-8aeb-c9044908c684.png)